### ISSUE TITLE
Is interactive mode works ?

### ISSUE THREAD

**Issue by herveleclerc** [Azure/InnovationEngine#137]
Is the feature active because when i try to use it i have only this output

```bash
./bin/ie interactive tutorial.md
Execute a document in interactive mode.
```

**Comment by mbifeld** [comment]
Thank you for your question!  No, this feature is currently a work in progress.  I see that the documentation improperly says that this is available, that will be updated shortly - apologies for that.

**Comment by herveleclerc** [comment]
Maybe
Can be a flag which allows you to say stop on each line of markdown or only on the blocks of code which will be used
thank for your hard work :)

**Comment by mbifeld** [comment]
Related: #157

**Comment by vmarcella** [comment]
Interactive mode was implemented in #165 and is available to use in our [latest release](https://github.com/Azure/InnovationEngine/releases/tag/v0.1.1) or by building the Innovation Engine by source!
